src/doc/src/reference/config.md

```diff
@@ -1257,6 +1257,19 @@ Controls whether or not extra detailed messages are displayed by Cargo.
 Specifying the `--quiet` flag will override and disable verbose output.
 Specifying the `--verbose` flag will override and force verbose output.
 
+#### `term.warnings`
+* Type: string
+* Default: "warn"
+* Environment: `CARGO_TERM_WARNINGS`
+
+Overrides how warnings are handled, including warnings that come from `rustc`.
+
+The `--warnings=` CLI option overrides this configuration value.
+
+* `warn` (default): warnings are displayed and do not fail the operation.
+* `error`: if any warnings are encountered an error will be emitted at the of the operation.
+* `ignore`: warnings will be silently ignored.
```

Comment:
How come you did `error` and `ignore` rather than `deny`/`forbid` and `allow`?

On one hand, the semantics are slightly different than the levels but on the other hand, they are familiar, predictable terms

Reply:
It seemed clearer what the behavior would be. If we do `deny` for "emit an error" and `allow` for "ignore warnings" what do we call the default "warn" option?

Maybe `deny`/`warn`/`allow` ?